store/src/commonMain/kotlin/com/dropbox/external/store5/impl/ShareableLruCache.kt

```diff
@@ -0,0 +1,123 @@
+@file:Suppress("UNCHECKED_CAST")
+
+package com.dropbox.external.store5.impl
+
+import co.touchlab.stately.collections.IsoMutableMap
+import co.touchlab.stately.isolate.IsolateState
```

Comment:
Maybe we can also use AtomicFU like in cache instead of having two libs.

Reply:
Going to remove this implementation

Reply:
Removed Stately!